politeiad/backendv2/tstorebe/plugins/pi/cmds.go

```diff
@@ -29,6 +30,126 @@ const (
 	dataDescriptorBillingStatus = pluginID + "-billingstatus-v1"
 )
 
+// cmdSummary returns the pi summary of a proposal.
+func (p *piPlugin) cmdSummary(token []byte) (string, error) {
```

Comment:
Order the commands in this file using the same order that they occur in the plugin API.